_posts/2019-01-25-updates.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Welcome to the New Way
+date: 2019-01-27  08:00:00
+categories: updates
+---
+__STATUS__:
+- current version: __3.60.0__
+- next scheduled release: __2/13/2019__
+
+Welcome to a new year, a new release, and a new set of development goals for the Ilios Project in the coming year! Key features identified as target milestones for our 2019-20 roadmap include:
```

Comment:
Starting the sentence with "Welcome to a new year" and ending it with "in the coming year!" is probably worth tweaking too:

```suggestion
Welcome to a new year, a new release, and a new set of development goals for the Ilios Project! Key features identified as target milestones for our 2019-20 roadmap include:
```